rd_ui/app/views/query.html

```diff
@@ -110,7 +110,7 @@ <h4 class="modal-title">Query Archive</h4>
               <span class="text-muted">Data Source</span>
               <select ng-disabled="!isQueryOwner" ng-model="query.data_source_id" ng-change="updateDataSource()"
                       ng-options="ds.id as ds.name for ds in dataSources"></select>
-
+              <a ng-if="dataSource.options.doc_url" ng-href={{dataSource.options.doc_url}}>{{dataSource.type_name}} documentation</a>
```

Comment:
As I'm not sure how I feel about this, can you hide it behind a feature flag (with default being off)?

(check how I did it for the [Manage Permissions](https://github.com/getredash/redash/blob/master/rd_ui/app/views/query.html#L128) button)

Thanks.